Northstar.Client/mod/resource/ui/menus/mod_settings.menu

```diff
@@ -0,0 +1,511 @@
+			"classname"					"ModButton"
+			"scriptID"					"12"
+			"pin_to_sibling"			"BtnMod12"
+			"pin_corner_to_sibling"		"TOP_LEFT"
+			"pin_to_sibling_corner"		"BOTTOM_LEFT"
+			"navUp"						"BtnMod12"
+			"navDown"					"BtnMod14"
+		}
+		"BtnMod14"
+		{
+			"ControlName"				"CNestedPanel"
+			"classname"					"ModButton"
+			"tall"						"45"
+			"wide"						"1200"
+			"controlSettingsFile"		"resource/UI/menus/panels/mod_setting.res"
+			"classname"					"ModButton"
+			"scriptID"					"13"
+			"pin_to_sibling"			"BtnMod13"
+			"pin_corner_to_sibling"		"TOP_LEFT"
+			"pin_to_sibling_corner"		"BOTTOM_LEFT"
+			"navUp"						"BtnMod13"
+			"navDown"					"BtnMod15"
+		}
+		"BtnMod15"
+		{
+			"ControlName"				"CNestedPanel"
+			"classname"					"ModButton"
+			"tall"						"45"
+			"wide"						"1200"
+			"controlSettingsFile"		"resource/UI/menus/panels/mod_setting.res"
+			"classname"					"ModButton"
+			"scriptID"					"14"
+			"pin_to_sibling"			"BtnMod14"
+			"pin_corner_to_sibling"		"TOP_LEFT"
+			"pin_to_sibling_corner"		"BOTTOM_LEFT"
+			"navUp"						"BtnMod14"
+			"navDown"					"BtnMod1"
+		}
+		// //////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+		"FilterPanel"
+		{
+			"ControlName"				"RuiPanel"
+			"wide"						"1220"
+			"tall"						"112"
+			//"xpos"						"-8"
+			"classname"					"FilterPanelChild"
+			"rui"						"ui/knowledgebase_panel.rpak"
+			"visible"					"1"
+			"zpos"						"-1"
+			"pin_to_sibling"			"FilterButtonsRowAnchor"
+			"pin_corner_to_sibling"		"TOP_LEFT"
+			"pin_to_sibling_corner"		"TOP_LEFT"
+		}
+		"LabelDetails"
+		{
+			"ControlName"				"RuiPanel"
+			"tall"						"695"
+			"wide"						"1220"
+			"xpos"						"10"
+			"ypos"						"10"
+			"pin_to_sibling"			"ButtonRowAnchor"
+			"pin_corner_to_sibling"		"TOP_LEFT"
+			"pin_to_sibling_corner"		"TOP_LEFT"
+			"rui"						"ui/knowledgebase_panel.rpak"
+			"wrap"						"1"
+			"visible"					"1"
+			"zpos"						"-1"
+		}
+		"BtnSearchLabel"
+		{
+			"ControlName"				"RuiButton"
+			"InheritProperties"			"RuiSmallButton"
+			"labelText"					"#SEARCHBAR_LABEL"
+			"textAlignment"				"west"
+			"classname"					"FilterPanelChild"
+			"wide"						"500"
+			"xpos"						"-23"
+			"ypos"						"-16"
+			"wrap"						"1"
+			"visible"					"1"
+			"zpos"						"0"
+			"pin_to_sibling"			"FilterButtonsRowAnchor"
+			"pin_corner_to_sibling"		"TOP_LEFT"
+			"pin_to_sibling_corner"		"TOP_LEFT"
+		}
+		"BtnModsSearch"
+		{
+			"ControlName"					"TextEntry"
+			"classname"						"FilterPanelChild"
+			"zpos"							"100"	// This works around input weirdness when the control is constructed by code instead of VGUI blackbox.
+			"xpos"							"-400"
+			"ypos"							"-5"
+			"wide"							"390"
+			"tall"							"30"
+			"textHidden"					"0"
+			"editable"						"1"
+			"font"							"Default_21"
+			"allowRightClickMenu"			"0"
+			"allowSpecialCharacters"		"1"
+			"unicode"						"0"
```

Comment:
Consider setting `unicode` to 1 to allow unicode search, like modlist menu (#401 )?

Reply:
resolved in b03c018